### DISCUSSION TITLE
Welcome Back, I don't know if this will be of any asistqnce but...

### DISCUSSION THREAD

**Discussion by hdstead** [rsjaffe/MIDI2LR#769] [General]
![Screenshot 2024-01-05 093530](https://github.com/rsjaffe/MIDI2LR/assets/49657532/719df437-e0ef-41bc-af75-d7893fc0fe81)
![Screenshot 2024-01-05 093530](https://github.com/rsjaffe/MIDI2LR/assets/49657532/2a5d2979-6afa-4861-9452-a1bc93e2f9ee)

This is a free AI offering for projects such as yours. My apologies in advance if this is old news. All the best to one and throughout 2025.

**Comment by antzpantz** [comment]
Spam.

**Comment by hdstead** [comment]
This was NOT spam.  The image showed that Microsoft offers free access to
software developers. And this group, rsjaffe/MIDI2LR should qualify.

On Sat, Jan 6, 2024 at 1:50 AM antzpantz ***@***.***> wrote:

> Spam.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rsjaffe/MIDI2LR/discussions/769#discussioncomment-8029622>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL23NPDZ6ZAWUYGSBEMC6UDYNEM7BAVCNFSM6AAAAABBPG5XPKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DAMRZGYZDE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
















*                                ''~`                             ( o o
)+---------------oooO--(_)--Oooo.----------------------------------+|| Doug
Stead,  Chairman,|Tri-M Group of Companies|www.Tri-M.Com
<http://www.Tri-M.Com>
 |                                                         |
"Self-deprecation is the sincerest form of flattery!"|               |
 |                     .oooO |                         (  )       Oooo.
 +---------------------\ (---- (
)------------------------------------------+
\_)    ) /                                     (_/*

**Comment by rsjaffe** [comment]
Thanks. I looked into it and I do qualify for free use. We'll see how useful (or not) it is. I tried JetBrains AI for a couple of days. It provided some useful refactoring advice but also included some serious errors in its recommendations. I'm really looking for lua help. It appears that the Microsoft service has some limited lua capabilities. We'll see.

**Comment by hdstead** [comment]
Thanks for the reply. I wish I had programming abilities to help. I tried
to learn Pascal at university back in the day, but my abilities to code
fell well short of my interest in technology. My communication skills could
be better; as I now see, my post had no reference to why the screenshots I
somehow put in twice might be beneficial.   AI may one day make your skills
as easy as setting errors off - do what I think.
All the best to you and your team, from all of us who benefit from your
effort.

Cheers,  Doug



On Sat, Jan 6, 2024 at 7:59 PM rsjaffe ***@***.***> wrote:

> Thanks. I looked into it and I do qualify for free use. We'll see how
> useful (or not) it is. I tried JetBrains AI for a couple of days. It
> provided some useful refactoring advice but also included some serious
> errors in its recommendations. I'm really looking for lua help. It appears
> that the Microsoft service has some limited lua capabilities. We'll see.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rsjaffe/MIDI2LR/discussions/769#discussioncomment-8034665>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL23NPFCPLFVB2CMUL6KELDYNIMSHAVCNFSM6AAAAABBPG5XPKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DAMZUGY3DK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
















*                                ''~`                             ( o o
)+---------------oooO--(_)--Oooo.----------------------------------+|| Doug
Stead,  Chairman,|Tri-M Group of Companies|www.Tri-M.Com
<http://www.Tri-M.Com>
 |                                                         |
"Self-deprecation is the sincerest form of flattery!"|               |
 |                     .oooO |                         (  )       Oooo.
 +---------------------\ (---- (
)------------------------------------------+
\_)    ) /                                     (_/*
